src/app/modules/main/activity_center/controller.nim

```diff
@@ -140,3 +140,21 @@ proc getMessageById*(self: Controller, chatId, messageId: string): MessageDto =
   if(err.len > 0):
     return MessageDto()
   return message
+
+proc setActiveNotificationGroup*(self: Controller, group: int) =
+  self.activityCenterService.setActiveNotificationGroup(ActivityCenterGroup(group))
+  self.activityCenterService.resetCursor()
+  let activityCenterNotifications = self.activityCenterService.getActivityCenterNotifications()
+  self.delegate.resetActivityCenterNotifications(activityCenterNotifications)
+
+proc getActiveNotificationGroup*(self: Controller): int =
+  return self.activityCenterService.getActiveNotificationGroup().int
+
+proc setActivityCenterReadType*(self: Controller, readType: int) =
+  self.activityCenterService.setActivityCenterReadType(ActivityCenterReadType(readType))
```

Comment:
Usually the module is the one converting the plain values to complex types, but it's not a huge deal

Reply:
fixed